hnn_core/parallel_backends.py

```diff
@@ -59,12 +59,18 @@ def _gather_trial_data(sim_data, net, n_trials, postproc):
 
     for idx in range(n_trials):
         dpls.append(sim_data[idx][0])
-        spikedata = sim_data[idx][1]
-        net.cell_response._spike_times.append(spikedata[0])
-        net.cell_response._spike_gids.append(spikedata[1])
+        sim_data_trial = sim_data[idx][1]
+        net.cell_response._spike_times.append(sim_data_trial[0])
+        net.cell_response._spike_gids.append(sim_data_trial[1])
         net.cell_response.update_types(net.gid_ranges)
-        net.cell_response._vsoma.append(spikedata[3])
-        net.cell_response._isoma.append(spikedata[4])
+        net.cell_response._vsoma.append(sim_data_trial[3])
+        net.cell_response._isoma.append(sim_data_trial[4])
+        for arr_name, arr in net.rec_array.items():
+            # voltages is a n_trials x n_contacts x n_samples array
+            net.rec_array[
```

Comment:
now `rec_array` is an instance ... confusing!

Reply:
`net.rec_array` is the dict of `ExtracellularArray`s. I'm not sure I follow what you mean by it being an 'instance' here?

Reply:
oh I see, indeed